tensorrt_llm/_torch/models/modeling_hunyuan_dense.py

```
@@ -0,0 +1,602 @@
+            max_position_embeddings=config.max_position_embeddings,
+            bias=config.attention_bias,
+            # rotary_emb=HunYuanTRTRotaryEmbedding(config),
+            # pos_embd_params=None,
+            pos_embd_params=pos_embd_params,
+            rope_fusion=not self.use_qk_norm,
+            layer_idx=layer_idx,
+            dtype=config.torch_dtype,
+            config=model_config,
+        )
+        if self.use_qk_norm:
+            if config.head_dim != None:
```

Ruff (0.12.2)

Comparison to `None` should be `cond is not None`

Replace with `cond is not None`

(E711)